lib/mix/test/mix/tasks/escript_test.exs

```diff
@@ -280,6 +302,57 @@ defmodule Mix.Tasks.EscriptTest do
     end)
   end
 
+  test "generate escript with distinct path for each application" do
+    in_fixture("escript_test", fn ->
+      Mix.Project.push(Escript)
+
+      Mix.Tasks.Escript.Build.run([])
+      assert_received {:mix_shell, :info, ["Generated escript escript_test with MIX_ENV=dev"]}
+      assert System.cmd("escript", ["escript_test", "--app-paths"]) == {"{true, true, true}\n", 0}
+    end)
+  end
+
+  test "generate escript without priv" do
```

Comment:
Instead of adding a new test for checking the default behaviour, we should just check this in one of the tests above, exactly because these tests can be very expensive in the long term.

Reply:
Done!